app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.kt

```diff
@@ -236,7 +237,8 @@ import javax.inject.Singleton
         AddBenchStatusOnBusStop(),
         AddBenchBackrest(),
         AddTrafficSignalsButton(),
-        AddPostboxRoyalCypher()
+        AddPostboxRoyalCypher(),
+        AddPoliceType()
```

Comment:
I'd move it a bit up in this list (which determines the quest priority), as the collected data seem more useful to me than those of some other quests. Maybe below `AddBoardType`?

Reply:
I was unsure of its importance, that's why I just put it there. But I trust you, I'll move it below AddBoardType 😉